data-loading/src/test/java/no/sikt/nva/data/report/api/etl/SingleObjectDataLoaderTest.java

```diff
@@ -66,18 +68,31 @@ void setup() {
     @Test
     void shouldFetchNviIndexDocumentAndTransformToCsvInExportBucket() throws IOException {
         var testData = new TestData();
-        var event = setupExistingIndexDocumentAndCreateUpsertEvent(testData);
+        var event = setupExistingNviIndexDocumentAndCreateUpsertEvent(testData);
         handler.handleRequest(event, context);
         var expected = testData.getNviResponseData();
         var actual = attempt(() -> sortResponse(CSV, getActualPersistedFile(ReportType.NVI), PUBLICATION_ID,
                                                 CONTRIBUTOR_IDENTIFIER)).orElseThrow();
         assertEquals(expected, actual);
     }
 
+    @ParameterizedTest
+    @EnumSource(names = {"AFFILIATION", "CONTRIBUTOR", "FUNDING", "IDENTIFIER", "PUBLICATION"})
+    void shouldFetchPublicationIndexDocumentAndTransformToFiveCsvFilesInExportBucket(ReportType reportType)
```

Comment:
`shouldFetchPublicationIndexDocumentAndTransformToKnownReportTypeAsCsvInExportBucket`

Reply:
You might want to check that you're not producing anything else in addition to these, which may occur when someone decides we need e.g. "licence" as a report type